plugins/rikki/heroeslounge/classes/matchmaking/Swiss.php

```diff
@@ -21,6 +21,12 @@ public function createMatches($pairings, $div) {
         $timeoffset = ($pairings[0][0]->region_id - 1)*36000;
         $schedule_date = date('Y-m-d H:i:s', strtotime('next sunday 21:55')+$timeoffset);
         $tbp = date('Y-m-d H:i:s', strtotime('+1 weeks sunday 21:55')+$timeoffset);
+        
+        // if it's the last round, games must be played until next friday (+5 days)
```

Comment:
Nit: "If it's the last round, games must be played until Friday instead of Sunday."